natives.json

```diff
@@ -56877,11 +56877,11 @@
 			"comment": "",
 			"params": [
 				{
-					"type": "Any",
+					"type": "int",
 					"name": "p0"
 				},
 				{
-					"type": "Any*",
+					"type": "int*",

```

Comment:
This is a gamer/network handle (a struct) so it should stay ``Any*``. I've seen too much bad code where people incorrectly use gamer handle stuff due to the params often being ``int*``.

Reply:
My bad. I took a look again and this was an error from my side.